kolibri/plugins/device/assets/src/views/DeviceSettingsPage/index.vue

```diff
@@ -872,6 +882,18 @@
         }
         return '';
       },
+      updateLimitForAutodownload() {
+        this.limitForAutodownload = this.toBytes(this.limitForAutodownloadInput);
+      },
+      updateLimitForAutodownloadInput() {
+        this.limitForAutodownloadInput = this.toGigabytes(this.limitForAutodownload);
+      },
+      toBytes(gigabytes) {
+        return parseInt(Math.round(gigabytes * 10 ** 9));
+      },
+      toGigabytes(bytes) {
+        return parseInt(bytesForHumans(bytes).replace(/[^0-9.]/g, ''));
```

Comment:
Any reason not to just do a pure calculation here?
```suggestion
        return parseInt(Math.round(bytes / 10 ** 9));
```

Reply:
No particular reason actually. I was just caught up within the two worlds of reuse existing functionality or new :), but pure calculation provide more precision I guess. I will make the change